crawl-ref/source/l-travel.cc

```diff
@@ -112,6 +112,28 @@ LUAFN(l_waypoint_delta)
     return 2;
 }
 
+/*** Set a numbered waypoint.
+ * Uses player-centered coordinates.
```

Comment:
I agree with you that player-centered reads best, but its neighbours use "player centered". They should match, either by making this like the existing comments, or fixing the existing comments.